packages/core/docs/package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "dev": "vuepress dev",
     "build": "vuepress build",
```

Comment:
add to build please ;)

Reply:
Added and generator works well, but only if dependencies inside `docs` are installed via `npm install` 😓  
Idk why but looks like `yarn` has some problems with required dependencies.